cmd/soroban-rpc/docker/Makefile

```diff
@@ -0,0 +1,18 @@
+SUDO := $(shell docker version >/dev/null 2>&1 || echo "sudo")
+
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+BUILD_DATE := $(shell date -u +%FT%TZ)
+
+TAG ?= stellar/stellar-soroban-rpc:$(SOROBAN_RPC_VERSION)
+
+STELLAR_CORE_VERSION ?= 19.10.1-1310.6649f5173.focal~soroban
+SOROBAN_RPC_VERSION ?= 0.8.0-21
```

Comment:
Hardcoding these here would force us to update these, or to have us invoke this makefile with the "correct" values.
I think that optionally providing a command-line override is perfectly fine, but these should be derived from the repositories themselves. 
in concrete actions, I'm thinking about the following:
1. find the latest soroban-rpc release tag in the soroban-tools repository. extract the version from there.
2. Use the above tag ( i.e. commit hash ), to extract the go.mod file. This would tell you the go monorepo hash. From there you could get the stellar core version and use it here.

If you need help figuring out how to extract that information, we've done a similar thing in the quickstart repository ( scripts dir )

Reply:
This could work but there are some challenges we'll need to address I think. Currently core builds are "promoted" manually. We can also have multiple builds (and packages) use the same commit and version, they can use different build flags for example. With that we'd need a way to pick which exact build to use.

I was also thinking about getting build tags from the docs but that's also not trivial - what if we want to build images before releasing them?

Reply:
while it's true that there are multiple builds for the same hash, these would eventually narrow down to a single build that you'd like to pick. 
Adding another reference is a guaranteed way for us to "miss" the release date.. I'd like to avoid doing that.